auto_dev/data/templates/protocols/protodantic.jinja

```diff
@@ -1,3 +1,5 @@
+from __future__ import annotations
+
```

Comment:
This DOES NOT resolve forward references on pydantic models, even tho it apparently works for python itself (since you no longer need to use a string-based reference for nested or self-referential types)